provider/cldr/src/lib.rs

```diff
@@ -50,9 +50,12 @@ use transform::locale_canonicalizer::likely_subtags::LikelySubtagsProvider;
 use transform::plurals::PluralsProvider;
 use transform::time_zones::TimeZonesProvider;
 
-pub fn create_exportable_properties_provider<T: DataMarker>(
+#[cfg(not(feature = "icu_list"))]
+type ListProvider = PluralsProvider; // we can't cfg-exclude part of the bound, but we can do this...
```

Comment:
I approve of this hack.  Hopefully we can remove the `cfg` soon, once BurntSushi delivers 0.2...